src/entity/point-of-sale/point-of-sale.ts

```diff
@@ -44,6 +44,10 @@ export default class PointOfSale extends BaseEntity {
   @ManyToOne(() => User, { nullable: false, eager: true })
   public owner: User;
 
+  @OneToOne(() => User, { nullable: false })
```

Comment:
OneToOne? So we are sure that every POS must have a unique user?

Reply:
I think it is good practice to do it that way, so you can always trace back who (or in this case: what) performed a certain action. If you use the same user everywhere, you cannot distinguish devices.